test/controllers/questions_controller_test.rb

```diff
@@ -1,22 +1,30 @@
+# frozen_string_literal: true
+
 require './test/test_helper'
 
 class QuestionsControllerTest < ActionDispatch::IntegrationTest
-
   class AuthenticatedTest < ActionDispatch::IntegrationTest
+    def expect_successful_response(expected_response)
+      assert_response :success
+      assert expected_response == response.body, "\nExpected:\n#{expected_response} \n\nActual:\n#{response.body.to_json}"
+    end
 
-    test "returns HTTP 200 and JSON response containing all public questions for" do
+    test 'tenant usage 0 when newly created' do
```

Comment:
I felt this was a good thing to separate from the other tests.

Reply:
Good to have this test, but I guess it'd be better placed at [the model's test file](https://github.com/piacsek/q_and_a/blob/main/test/models/tenant_test.rb). WDYT?

Reply:
Makes sense, either is good to me - I just quickly extracted it from a test in this file (it was only the first half, so I figured it was important to be here lol)